### ISSUE TITLE
Postfix lens: not possible (or not obvious) to edit space separated values

### ISSUE THREAD

**Issue by minfrin** [hercules-team/augeas#485]
In the /etc/postfix/main.cf config file, many of the parameters are space separated, for example:

`smtp_milter = A B C`

Currently there seems to be nothing in the lens that allows augeas to manipulate the value of smtp_milter, most specifically to add an additional entry:

`smtp_milter = A B C D`

Or remove a given entry:

`smtp_milter = A C`

The lens needs to be updated to support space separated config parameters.

**Comment by raphink** [comment]
Indeed, the value accepts spaces, which means "A B C" is parsed as a single node. Changing this would mean:

* treating `smtp_milter` as a special key with array value
* breaking compatibility on this lens
* are there other keys that should behave like an array in this lens?
